.github/CODEOWNERS

```diff
@@ -91,7 +91,7 @@ test/boost/mutation_reader_test.cc @denesb
 test/boost/querier_cache_test.cc @denesb
 
 # PYTEST-BASED CQL TESTS
-test/cql-pytest/* @nyh
+test/cql_pytest/* @nyh
```

Comment:
> 1. It makes backports of cql-pytest tests a mess (at least, I think it does?).

It shouldn't, I think `git` is smart enough to help here.
(how often do we have to backport tests anyway?)

> Alternatively, why can't mypy understand the import hacks? Is that a mypy bug? Isn't it 100% legal Python to change the import path?

I think the problem is that imports were resolved by modifying `sys.path` in runtime. Since mypy and other static code analysis tools work "before" run-time (at which point the value of `sys.path` is unknown), they get bugged out.

Also, this way of resolving imports seems to be viral: as soon as one module depends on the sys.path hack, all modules that want to import that module must also do the sys.path hack. So it makes the lives of developers miserable too.